docs/en/latest/plugins/server-info.md

```diff
@@ -89,18 +95,17 @@ $ curl http://127.0.0.1:9090/v1/server_info -s | jq .
 }
 ```
 
-The APISIX Dashboard will collects server info in etcd, so you may also try to check them through Dashboard.
+:::tip
+
+You can also view the server info report through the [APISIX Dashboard](/docs/dashboard/USER_GUIDE).
+
+:::
 
 ## Disable Plugin
 
-Remove `server-info` in the plugin list of configure file `conf/config.yaml`.
+To disable the Plugin, you can remove `server-info` from the list of Plugins in your configuration file:
 
-```
-plugins:                          # plugin list
-  - example-plugin
-  - limit-req
-  - node-status
-  - jwt-auth
-  - zipkin
-  ......
+```yaml title="conf/config.yaml"
```

Comment:
Could you explain about `yaml title="conf/config.yaml"` part? I can't really get the point of using it.

Reply:
It will show up on the site like this. This will avoid us having to comment that the file in conf/config.yaml as it has been done in a lot of places.

<img width="847" alt="Screenshot 2022-03-31 at 12 29 54 PM" src="https://user-images.githubusercontent.com/49474499/160996736-d0594192-dcad-452c-af15-cff9ec6c1695.png">

Reply:
Also see: https://docusaurus.io/docs/next/markdown-features

Reply:
cool